Solinea/goldstone-server
# configure_stack cleanup, docker images cleanup, new RDO-kilo instance

To test:

- download the updated one referenced in HACKING.md, double-click it to import
- shut down and rename your RDO-kilo virtualbox image
- ensure that it is named `RDO-kilo`, and start it
- `docker rm -f $(docker ps -qa)`
- `docker rmi goldstoneserver_gsdb goldstoneserver_gsappdev`
- `docker rmi -f $(docker images | grep '0.8.5-SNAPSHOT' | awk '{print $3}')`
- `bin/start_dev_env.sh`
- `docker exec -it goldstoneserver_gsappdev_1 bash`

Then, inside the docker container:
- `fab -f post_install.py -H 172.24.4.100 configure_stack:172.24.4.1`

Once complete, log into goldstone and verify that data starts flowing within about 5 minutes.

closes: #322
